### ISSUE TITLE
Add configuration options for plugins

### ISSUE THREAD

**Issue by SirPepe** [TurbineCSS/Turbine#66]
For example to tell boxshadow to stop adding these ugly filter shadows in IE. My proposal:

```
@configure
    boxshadow:noie // "No IE" option
    someotherplugin:something, somethingelse // Multiple options
```

**Comment by SirPepe** [comment]
Maybe simply add this to `@turbine`?

**Comment by SirPepe** [comment]
Implemented in dev in 9d49a48150e78b0378f4
